src/9on12Resource.cpp

```diff
@@ -639,6 +639,8 @@ namespace D3D9on12
         memset(&m_TranslationLayerCreateArgs, 0, sizeof(m_TranslationLayerCreateArgs));
     }
 
+    std::unordered_map<Resource*, std::vector<LockRange>> g_umapDiscardResources;

```

Comment:
Was thinking about this some more last night and talked with Jesse a little too, and rather than making this a global it should probably be a member of D3D9on12::Device.

Reply:
Sure thing; fix incoming.